src/App.tsx

```diff
@@ -1,7 +1,49 @@
 import React from 'react';
+// import classNames from 'classnames';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type Props = {
+};
+
+type State = {
+  pressedKey: string;
+};
+
+export class App extends React.Component<Props, State> {
+  state = {
```

Comment:
I think, it's not necessary to create empty Props. You can use just {} in generic))
```suggestion
type State = {
  pressedKey: string;
};

export class App extends React.Component<{}, State> {
  state = {
```

Reply:
ok